presto-parquet/src/main/java/com/facebook/presto/parquet/reader/PageReader.java

```diff
@@ -60,12 +77,15 @@ public DataPage readPage()
         }
         DataPage compressedPage = compressedPages.removeFirst();
         try {
+            long firstRowIndex = getFirstRowIndex(pageIndex++, offsetIndex);
```

Comment:
could we move `pageIndex = pageIndex + 1` out of the function call?

Reply:
sure